contracts/SSVNetwork.sol

```diff
@@ -126,14 +126,19 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
 
         // Operator[] memory operators;
         bytes32 clusterId = _getOrCreateCluster(operatorIds);
+        bytes32 decodedValidator = keccak256(publicKey);
+        bytes32 decodedPod = keccak256(abi.encodePacked(msg.sender, clusterId));
```

Comment:
All things involving Validator with keccak256 that has repeats i made predefined vars which helped to bring down gas